language-server/modules/langserver-core/src/test/resources/codeaction/create-function/source/create_function_in_local_var2.bal

```diff
@@ -0,0 +1,3 @@
+public function main() {
+    int var1 = createInt();
+}
```

Comment:
```suggestion
}

```
Shall we add a new line here

Reply:
This is intentional. This test case was added to verify that the code action does not add a new line when the user have not added a new line